### PR TITLE
GFORMS-1362 Remove ' - HMRC' suffix in email subject

### PR DESCRIPTION
**Pull request by girishkamat**

### CODE REVIEW

File: app/uk/gov/hmrc/gform/config/ConfigModule.scala [hunk 1]
```diff
@@ -101,7 +101,7 @@ class ConfigModule(val context: ApplicationLoader.Context, playBuiltInsModule: P
       } yield s"$url$projectId.js",
       trackingConsentSnippet = new hmrcTrackingConsentSnippet(new TrackingConsentConfig(playConfiguration)),
       emailAuthStaticCodeEmails =
-        getOptionalNonEmptyCIStringList(playConfiguration.getOptional[String]("emailAuth.staticCodeEmails"))
+        getOptionalNonEmptyCIStringList(playConfiguration.getOptional[String]("emailAuth.staticCodeEmails:q"))
```
Comment (josef-vlach) [inline app/uk/gov/hmrc/gform/config/ConfigModule.scala:104]: This looks like someone trying to quit vim error?

Comment (girishkamat) [reply app/uk/gov/hmrc/gform/config/ConfigModule.scala]: he he...good spot and good guess :D

Comment (girishkamat) [reply app/uk/gov/hmrc/gform/config/ConfigModule.scala]: I will be doing another quick fix Jonathan asked for, spacing in the text shown on confirm code page

Comment (girishkamat) [reply app/uk/gov/hmrc/gform/config/ConfigModule.scala]: Fix has been added

**Review by josef-vlach — APPROVED** [review]

**Pull request merged**